### ISSUE TITLE
Default LOG gets not cleared in non development environment

### ISSUE THREAD

**Issue by manuelselbach** [helhum/typo3-config-handling#1]
The default values for `$GLOBALS['TYPO3_CONF_VARS']['LOG']` only get reseted in `Helhum\TYPO3\ConfigHandling\ConfigLoader::populate()`, thus the default `LOG` settings are applied for all other environments than development.

**Comment by helhum** [comment]
can you articulate your expectations vs. the actutal behavior?

**Comment by manuelselbach** [comment]
In Development context the default configuration stored in `$GLOBALS['TYPO3_CONF_VARS']['LOG']` will be reseted, thus there is a clean configuration and the configuration from the file will be strict.

In all other contexts this behavior is different, as the configuration will not be reseted.

I would expect the same behavior for every context, thus the default configuration should be either be reseted in every context or should stay as it is.

**Comment by mkarulin** [comment]
Temporary solution is to use "__UNSET". Thanks Helmut.

```
LOG:
    writerConfiguration:
        '%const(TYPO3\CMS\Core\Log\LogLevel::EMERGENCY)%':
            TYPO3\CMS\Core\Log\Writer\PhpErrorLogWriter: {  }
       
        '%const(TYPO3\CMS\Core\Log\LogLevel::WARNING)%':
            TYPO3\CMS\Core\Log\Writer\PhpErrorLogWriter: {  }
            TYPO3\CMS\Core\Log\Writer\FileWriter: __UNSET

        '%const(TYPO3\CMS\Core\Log\LogLevel::DEBUG)%':
            TYPO3\CMS\Core\Log\Writer\PhpErrorLogWriter: {  }
    TYPO3:
        CMS:
            Core:
                Resource:
                    ResourceStorage:
                        writerConfiguration:
                             '%const(TYPO3\CMS\Core\Log\LogLevel::ERROR)%':
                                  TYPO3\CMS\Core\Log\Writer\PhpErrorLogWriter: {  }
                                  TYPO3\CMS\Core\Log\Writer\FileWriter: __UNSET
                                  TYPO3\CMS\Core\Log\Writer\DatabaseWriter: __UNSET
```

**Comment by helhum** [comment]
fixed with db497825d00b8b0bd6bbd9d0986c07d8324abd49

**Comment by helhum** [comment]
import changed with beac45b0a2137484d03c1c95c31a22c04f6eb197

```
imports:
    - { resource: 'DefaultConfiguration', type: typo3, exclude: [LOG] }
```
